app/controllers/job_messages_controller.rb

```diff
@@ -10,29 +10,30 @@ def show
 
   def get
     status = ActiveJob::Status.get(params[:job_id])
-    if status.working?
-      if status[:job_class]
-        flash_now(:notice, status[:job_class].show_status(status))
-      else #default x out of y message
-        flash_now(:notice, t('poll_job.working_message', progress: status[:progress], total: status[:total]))
-      end
+    if status.queued?
```

Comment:
I'm wondering if before this we should actually be checking for `status.nil?`; I don't think this has come up before, but it seems like this might be a good thing to check just to be safe.

Reply:
Yes, why not? Better safe than sorry 👍

Reply:
Actually, `status` will never be `nil` because `ActiveJob::Status.get` calls its `new` method which will always create a new object. If a job with `:job_id` doesn't exist already then no data will be read from `Storage` and the status will be empty instead of `nil`. In this case, it will be handled by the `else` block.

Reply:
This has been resolved by adding the ability for this method to render a 404 error if there is no status messages (it is empty not nil).